packages/ts/src/components/chord-diagram/style.ts

```diff
@@ -78,14 +74,10 @@ export const link = css`
   stroke: var(--vis-chord-diagram-link-stroke-color);
   stroke-opacity:  var(--vis-chord-diagram-link-stroke-opacity);
   transition: .1s fill-opacity;
-
-  &:hover {
-    fill-opacity: 1;
-  }
 `
 
-export const hoveredLink = css`
-  label: hovered;
+export const highlightedLink = css`
+  label: highlighted;
   fill-opacity: 0.9;
```

Comment:
I think we also need to add CSS variables to control the opacity of highlighted / dimmed nodes and links